src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -95,6 +99,19 @@
 	private AppSettings appSettings;
 	private Vertx vertx;
 
+	protected List<String> encoderBlockedStreams = new ArrayList<>();
+	private int numberOfEncoderNotOpenedErrors = 0;
+	protected int publishTimeoutStreams = 0;
+	private List<String> publishTimeoutStreamsList = new ArrayList<>();
+	
+	protected WebRTCVideoReceiveStats webRTCVideoReceiveStats = new WebRTCVideoReceiveStats();
+
+	protected WebRTCAudioReceiveStats webRTCAudioReceiveStats = new WebRTCAudioReceiveStats();
+	
+	
+	protected WebRTCVideoSendStats webRTCVideoSendStats = new WebRTCVideoSendStats();
+
+	protected WebRTCAudioSendStats webRTCAudioSendStats = new WebRTCAudioSendStats();
```

Comment:
What do you think about collecting these stat classes in a WebRTCStat class?
encoderBlockedStreams and other fields could be properties of this class.